parsers/IL.py

```diff
@@ -55,14 +55,29 @@ def flatten_list(_2d_list) -> list:
     return flatten_list(cleaned_list)
 
 
+def fetch_price(zone_key="IL", session=None, target_datetime=None, logger=None) -> dict:
+    """Fetch price from IEC table."""
+    response = get(IEC_PRICE)
+    soup = BeautifulSoup(response.content, "lxml")
+
+    price = soup.find("td", class_="ms-rteTableEvenCol-6")
+
+    return {
+        "zoneKey": zone_key,
+        "currency": "NIS",
+        "datetime": arrow.now(TZ).datetime,
+        "price": price.p.text,
```

Comment:
Could you cast this value to a float?
I think `float(price.p.text)` should suffice.